src/SDL_hints.c

```diff
@@ -83,13 +83,28 @@ static void SDLCALL CleanupHintProperty(void *userdata, void *value)
     SDL_free(hint);
 }
 
+static const char* GetHintEnvVar(const char *name)
+{
+    const char *ret = SDL_getenv(name);
```

Comment:
```suggestion
    const char *result= SDL_getenv(name);
```

We've switched to this common variable name for values returned from a function, FYI.